docs/user-guide/cli-installplugins.md

```diff
@@ -139,7 +141,7 @@ zowe plugins validate [plugin]
     |IBM Db2 Plug-in for Zowe CLI|`@zowe/db2-for-zowe-cli`|
     |IBM IMS Plug-in for Zowe CLI|`@zowe/ims-for-zowe-cli`|
     |IBM MQ Plug-in for Zowe CLI|`@zowe/mq-for-zowe-cli`|
-    |||
+    | Secure Credential Store Plug-in for Zowe CLI | `@zowe/secure-credential-store-for-zowe-cli` |   |||
```

Comment:
Recommendation: Don't add extra columns at the end in order to prevent any strange behavior. : )